src/components/common/ConfirmModal.tsx

```diff
@@ -58,12 +58,14 @@ const ConfirmModal = ({
   title,
   body,
   actionLabel,
+  isMap,
```

Comment:
As discussed, this will be resolved by [this ticket](https://www.notion.so/uwblueprintexecs/396370c5e09f4c28bc7f455a7e84c908?v=3751aa62a2594c74b7e0ecb363304c24&p=d7885c94b20c426cb1258dedac09dd12)